src/01_getting_started/03_state_of_async_rust.md

```diff
@@ -4,21 +4,22 @@ The asynchronous Rust ecosystem has undergone a lot of evolution over time,
 so it can be hard to know what tools to use, what libraries to invest in,
 or what documentation to read. However, the `Future` trait inside the standard
 library has recently been stabilized, and the `async`/`await` feature will
-follow shortly. The ecosystem as a whole is therefore in the midst of migrating
-to the newly-stabilized API, after which point churn will be significantly
-reduced.
+follow shortly.
 
-At the moment, however, the ecosystem is still undergoing rapid development
-and the asynchronous Rust experience is unpolished. Most libraries still
-use the 0.1 definitions of the `futures` crate, meaning that to interoperate
-developers frequently need to reach for the `compat` functionality from the
-0.3 `futures` crate. The `async`/`await` language feature is still new.
-Important extensions like `async fn` syntax in trait methods are still
-unimplemented, and the current compiler error messages can be difficult to
-parse.
+The ecosystem is currently in a state of migration from the 0.1 `futures` crate
+to the 0.3 `futures` crate. While this is ongoing, you may find that you will
+have to reach for the `compat` functionality of the 0.3 crate. You may also
+find that the `async`/`await` language feature is still new, so examples
+may take some time to pop up. There's still some language oddities, like
+`async fn` functions not working on traits and error messages are under constant
```

Comment:
Maybe rephrase this sentence. wdyt: "`async`/`await` is still under active development, and there are important improvements on the way, such as `async fn` in traits and diagnostics cleanup."